.github/workflows/codespell.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master,hypernext,next]
+  pull_request:
+    branches: [master,hypernext,next]
+
+jobs:
+  codespell:
+    name: Check for typos
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+      - name: Install codespell
```

Comment:
Maybe add a comment to explain why we do not use codespell-action with a link to your issue

Reply:
I did that in the commit msg, but ok, doing now as a comment too.

Reply:
Yeah, I find it easier to directly read a comment in the code. Going to `blame` is last resort!

Reply:
well, we are then more alike than different I must say ;)